roles/cloud-bootstrap/tasks/accounts.yml

```diff
@@ -0,0 +1,81 @@
+---
+- name: Ensure shared projects
+  os_project:
+    cloud: "{{ cloud }}"
+    name: "{{ item }}"
+  with_items:
+    - "{{ prod_project }}"
+    - "{{ nodepool_project }}"
+
+- name: Ensure user projects
+  os_project:
+    cloud: "{{ cloud }}"
+    name: "{{ item.username }}"
+  with_items: "{{ users }}"
+
+- name: Ensure groups
+  os_group:
+    cloud: "{{ cloud }}"
+    name: "{{ item }}"
+  with_items:
+    - "{{ prod_group }}"
+    - "{{ prod_admin_group }}"
+
+- name: Ensure users
+  os_user:
+    cloud: "{{ cloud }}"
+    name: "{{ item.username }}"
+    email: "{{ item.email }}"
+    default_project: "{{ item.username }}"
+  with_items: "{{ users }}"
+
```

Comment:
Should copy over the task that grants the user a role on their project, otherwise they won't be able to do anything.

Grab the task from https://github.com/BonnyCI/hoist/pull/168/files#diff-e0b563178351dc3acf8378b82947655eL18 and make sure any defaults used are ported over as well.